### PR TITLE
Updated opme to make things alot easiler

### PR DESCRIPTION
**Pull request by looperXD**
The Command /opme was only open to admins, which is kinda unfair.
I updated the command /opme so when a new player joins or a player comes back from griefing..etc they can op them self rather them screaming and yelling in the chat  "OP ME!, OP pls" or having the admins do /opall everytime when a new player joins.

I blocked the command to Imposter so when a Imposter try to use /opme they will not get op, but  de-opped if they aren't de-opped already.

### CODE REVIEW

**Comment by xDestroyer217** [comment]
Well, this does need testing.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java [hunk 1]
```diff
@@ -14,8 +15,13 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
...
         TFM_Util.adminAction(sender.getName(), "Opping " + sender.getName(), false);
         sender.setOp(true);
-        sender.sendMessage(TFM_Command.YOU_ARE_OP);
-
+            sender.sendMessage(TotalFreedomMod.YOU_ARE_OP);
+               if (!TFM_AdminList.isAdminImpostor(sender_p))
+               {
+               sender.sendMessage("You are a Imposter, You need to verify first");
+                sender.setOp(false);
+            sender.sendMessage(TotalFreedomMod.YOU_ARE_NOT_OP);
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: Why are you sending the NOT OP message? They are not op already, so this should not need to be happening.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java [hunk 2]
```diff
@@ -14,8 +15,13 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
     {
         TFM_Util.adminAction(sender.getName(), "Opping " + sender.getName(), false);
         sender.setOp(true);
-        sender.sendMessage(TFM_Command.YOU_ARE_OP);
-
+            sender.sendMessage(TotalFreedomMod.YOU_ARE_OP);
+               if (!TFM_AdminList.isAdminImpostor(sender_p))
+               {
+               sender.sendMessage("You are a Imposter, You need to verify first");
+                sender.setOp(false);
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: Should not have been set to OP True in the first place...

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java [hunk 3]
```diff
@@ -14,8 +15,13 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
     {
         TFM_Util.adminAction(sender.getName(), "Opping " + sender.getName(), false);
         sender.setOp(true);
-        sender.sendMessage(TFM_Command.YOU_ARE_OP);
-
+            sender.sendMessage(TotalFreedomMod.YOU_ARE_OP);
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: This should have a if not imposer wrapped around it, then have an else for if they are to send them the message that they cannot get OP

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java [hunk 4]
```diff
@@ -2,10 +2,11 @@
 
 import me.StevenLawson.TotalFreedomMod.TFM_Util;
 import org.bukkit.command.Command;
+import me.StevenLawson.TotalFreedomMod.*;
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: Why why why?????? This should NOT be needed, and the star imports from what I understand should only be used as an absolute last resort. This should not be needed either way.

**Comment by Wild1145** [comment]
Formatting is also required as it currently is not formatted correctly.

**Comment by JeromSar** [comment]
@looperXD As per the contributing guidelines, please make an issue describing the changes you'd like to see implemented before making a PR. This PR is on hold until you do so. :)

**Comment by Nobody-11** [comment]
I believe this is a bad idea due when people using a large W/E and got dropped, they can reaped themselves… @JeromSar @looperXD @StevenLawson @Wild1145

**Comment by JeromSar** [comment]
@Nobody-11 Please don't tag every developer.

**Comment by looperXD** [comment]
Updated.

**Comment by DaBoyTM** [comment]
@looperXD is there anyway you can possibly stop people from opping themselves after getting deopped from W/E Limits

**Comment by Wild1145** [comment]
@looperXD Please review my line comments as there are things you should change to make it run a lot smoother.

**Comment by Wild1145** [comment]
@looperXD Please take a look again and let us know if you plan to update it.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java (hunk 2 above)
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: Also, formatting.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java (hunk 3 above)
Comment (AlexFreedomMod) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: Ops will get the message below :/

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java [hunk 5]
```diff
@@ -14,8 +16,12 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
     {
         TFM_Util.adminAction(sender.getName(), "Opping " + sender.getName(), false);
         sender.setOp(true);
-        sender.sendMessage(TFM_Command.YOU_ARE_OP);
-
+            sender.sendMessage(TotalFreedomMod.YOU_ARE_OP);
+               if (!TFM_AdminList.isAdminImpostor(sender_p))
```
Comment (AlexFreedomMod) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: Should be 
if (TFM_AdminList.isAdminImposter(sender_p))

**Comment by Scuph** [comment]
@AlexFreedomMod no, its correct

**Comment by AlexFreedomMod** [comment]
I might (MIGHT) remake this.

**Pull request closed without merging**